lib/widgets/common_history_tools.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:history_manager/history_manager.dart';
+
+const _deactivated = Color(0xFFBDBDBD);
+
+const _redoText = 'REDO';
+
+const _undoText = 'UNDO';
+
+class CommonHistoryTools extends StatelessWidget {
```

Comment:
Document this class. There shouldn't be anything undocumented for any part of a library